docs/developer_guide/setup.md

```diff
@@ -0,0 +1,25 @@
+# Setting up MedModels
+
+MedModels leverages a combination of Python and Rust code. To contribute effectively, you'll need to set up a development environment that supports both languages.
+
+**Requirements:**
+
+- Python (3.9, 3.10, 3.11 or 3.12)
+- Rust compiler (follow instructions from [https://www.rust-lang.org/tools/install](https://www.rust-lang.org/tools/install))
+
+**Using the Makefile:**
+
+MedModels utilizes a `Makefile` to manage development tasks. Here's a breakdown of the available commands and their functionalities:
```

Comment:
We are missing this information that was included before. Do we want to get rid of it? We do not show how Makefile commands are run.

**Using the Makefile with your IDE:**

The provided `Makefile` can be integrated with most IDEs. Consult your IDE's documentation for instructions on adding custom build tasks based on Makefiles. This allows you to easily run commands like `make install-dev` or `make test` directly from within your IDE.

By following these steps and leveraging the `Makefile`, you'll have a development environment ready to contribute to the MedModels codebase!

Reply:
I removed this part because I didn't find it necessary to include it in the guide in the first place but I can add it back if you find it helpful.

Makefiles are practically always run using `make ..`, adding it to the IDE is just a convenience that is not necessary. IMO if a user wants to do it, go ahead but I wouldn't suggest it. That feels out of place in the medmodels guide